labeller/labeller.go

```diff
@@ -506,3 +511,12 @@ func (l *Labeller) podInNamespaces(pod *corev1.Pod) bool {
 	}
 	return false
 }
+
+func (l *Labeller) handleNoop(obj, oldObj *secscanv1alpha1.ImageManifestVuln) {
+	if _, ok := obj.GetLabels()["imagemanifestvuln.example"]; ok && !reflect.DeepEqual(obj.Spec, oldObj.Spec) {
```

Comment:
Should we probably change the label name to something more descriptive?
imagemanifestvuln.skipgc?
Is an annotation better for this use case?

Reply:
Done